recipes/sdl2_ttf/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "SDL2_ttf" %}
+{% set version = "2.0.14" %}
+{% set sha256 = "34db5e20bcf64e7071fe9ae25acaa7d72bdc4f11ab3ce59acc768ab62fe39276" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url:  https://www.libsdl.org/projects/SDL_ttf/release/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - toolchain
+    - sdl2
+    - freetype
+
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - sdl2
+    - freetype
```

Comment:
Please update this with a proper [pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/bfaae94924c29d89e2288f06835b6461f4bebf94/scripts/pin_the_slow_way.py#L37 ).